python/lsst/ts/xml/data/sal_interfaces/MTM2/MTM2_Commands.xml

```diff
@@ -256,4 +256,16 @@
       <Count>6</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_enableLutTemperature</EFDB_Topic>
+    <Description>Enable the calculation of the look-up table (LUT) for the temperature or not.</Description>
```

Comment:
Use temperature LUT to improve mirror shape.

Reply:
Improve the description in the commit: [cbee6f6](https://github.com/lsst-ts/ts_xml/pull/882/commits/cbee6f6bae6a6dad4b34c9e9a205489cdc85d0fc). Thanks!